File: assignment/may29/pro4.c
#include<stdio.h>
int main()
{
    int num;
    int even=2;
    printf("enter the num:");
    scanf("%d",&num);
    for( int i=1;i<=num;i++)
    {
        for(int j=1;j<=num-i;j++)
        {
            printf(" ");
        }
       for(int k=1;k<=i;k++)
       {
           printf("%d ",even);
           even = even + 2;
       }
       printf("\n",num);
    }
}

File: C_Workspace/python/june_11.py/pro_4.py
num=int(input("enter the number"))
rev=0
digit=0
while(num):
    digit=num%10
    rev=rev*10+digit
    num=num//10
print(rev)    
print("orginal number print")
org=0
digit=len(str(rev))
for i in range(digit):
    rem=rev%10
    org=org*10+rem
    rev=rev//10
print(org)    